Service/ZGWToXxllncZaakService.php

```diff
@@ -0,0 +1,499 @@
+<?php
+
+namespace CommonGateway\XxllncZGWBundle\Service;
+
+use App\Entity\Attribute;
+use App\Entity\Entity as Schema;
+use App\Entity\Gateway as Source;
+use App\Entity\ObjectEntity;
+use App\Entity\Synchronization;
+use App\Entity\Value;
+use CommonGateway\CoreBundle\Service\CallService;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\PersistentCollection;
+use Doctrine\Persistence\ObjectRepository;
+use Exception;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class ZGWToXxllncZaakService
+{
+    private EntityManagerInterface $entityManager;
+    private CallService $callService;
+    private SymfonyStyle $io;
+    private array $configuration;
+    private array $data;
+
+    private ObjectRepository $schemaRepo;
+    private ObjectRepository $sourceRepo;
+
+    private ?Source $xxllncAPI;
+    private ?Schema $xxllncZaakSchema;
+
+    /**
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        CallService $callService
+    ) {
+        $this->entityManager = $entityManager;
+        $this->callService = $callService;
+
+        $this->schemaRepo = $this->entityManager->getRepository(Schema::class);
+        $this->sourceRepo = $this->entityManager->getRepository(Source::class);
```

Comment:
I don't like doing this^ (just my opinion), but can't come up with a reason why we shouldn't do this right now.

Reply:
->getRepository('App:Entity');
@bbrands02